packages/tables/demo/stories/TableStory.tsx

```diff
@@ -147,7 +151,11 @@ export const TableStory: Story<IArgs> = ({
                   </Cell>
                 )}
                 {Object.keys(row).map((column, columnIndex) => (
-                  <Cell key={`${rowIndex}${columnIndex}`} isTruncated={isTruncated}>
+                  <Cell
+                    key={`${rowIndex}${columnIndex}`}
+                    isTruncated={isTruncated}
+                    hidden={isHidden && rowIndex === 2 && columnIndex === 0}
```

Comment:
Since the table is data-defined, I'd recommend just hiding all rows for story clarity.

Reply:
💥 [c33a847](https://github.com/zendeskgarden/react-components/pull/1514/commits/c33a847918a15f751b0ce425c5ae60c93ef3d771)